core/src/main/resources/hudson/model/User/sidepanel.jelly

```diff
@@ -31,7 +31,6 @@ THE SOFTWARE.
   <l:header />
   <l:side-panel>
     <l:tasks>
-      <l:task contextMenu="false" href="${rootURL}/asynchPeople/" icon="symbol-people" title="${%People}"/>
```

Comment:
This is an awkward case as the people view basically represents the list of users, of which an individual user is an entry. Unless using the built-in Jenkins' own user database, admins no longer have a users list with this change. We should probably add a `ManagementLink` before long to iterate over (actual) users.

Reply:
> Unless using the built-in Jenkins' own user database, admins no longer have a users list with this change.

They do not have such a list currently. `AsynchPeople` lists SCM users which are not in the security realm; and does not list users which _are_ in the security realm but have no user record on disk. It is simply useless for this purpose.

Reply:
> but have no user record on disk

Right, but anyone who's logged in once has such a record, which makes it a viable (if crappy and slow) entrypoint to getting to someone that's not directly entering the URL.